meerk40t/core/node/elem_line.py

```diff
@@ -136,4 +141,5 @@ def as_path(self):
         self.shape.stroke_width = self.stroke_width
         self.shape.linecap = self.linecap
         self.shape.linejoin = self.linejoin
+        self.shape.fillrule = self.fillrule
```

Comment:
I guess this is fine, lines are never filled though, but for consistency I guess... but if that's the case, why not rect and circle? (those are non-crossing and can't need a fill rule either).